example/CMakeLists.txt

```diff
@@ -0,0 +1,8 @@
+PROJECT(mciExample CXX)
+
+FIND_PACKAGE(ITK REQUIRED COMPONENTS ITKMorphologicalContourInterpolation ITKSmoothing ITKIOImageBase)
```

Comment:
`ITKMorphologicalContourInterpolation` -> `MorphologicalContourInterpolation`

we also need to add IO format modules to read files.

Please also add a test -- here is an example example :-P

  https://github.com/InsightSoftwareConsortium/ITKAnisotropicDiffusionLBR/blob/f3af35b5cb9388322c199d9c120bc369c5bb902d/examples/CMakeLists.txt#L19-L33